### ISSUE TITLE
StructuralRules.groupSequence()

### ISSUE THREAD

**Issue by PtrMan** [opennars/opennars#272]
Write a NAL test which hits [StructuralRules.groupSequence()](https://codecov.io/gh/opennars/opennars/src/master/src/main/java/org/opennars/inference/StructuralRules.java#L660)

**Comment by patham9** [comment]
Good idea to unit-test it. The list connector was experimental so far, if you see transformation rules that are missing, let me know.
